3.8/arangosearch-analyzers.md

```diff
@@ -513,6 +521,16 @@ attributes:
 - `memoryLimit` (integer): memory limit for query execution in bytes.
   (default is 1048576 = 1Mb) Maximum is 33554432U (32Mb)
 
+- `returnType` (string): Type of returned tokens.
```

Comment:
Tokens (plural) - does that mean the AQL query can return an array where each element is of the indicated type?

Reply:
No, it means, analyzer could emit arbitrary number of tokens.

Reply:
So `RETURN [1, 2]` will not work (result: `[ [ 1, 2 ] ]`) but `FOR val IN [1, 2] RETURN val` will? (result: `[1, 2]`)?
In other words, each top-level element of the result is a token, or am I getting this totally wrong?

Reply:
Actually RETURN [1,2] will work. All will depend on  `returnType` value. E.g. if it is string than token will be "[1,2]". And yes you are right each value emitted by RETURN node is considered as transformation function input. If it is array or object - it will be converted as one entity, not iterated.